### DISCUSSION TITLE
Clean up Build jobs?

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-tycho/tycho#291] [Ideas]
@mickaelistria  @akurtakov As I'm currently taking a deeper look at our [Jenkins](https://ci.eclipse.org/tycho/) due to [Bug 575285](https://bugs.eclipse.org/bugs/show_bug.cgi?id=575285) I have noticed we have some jobs that have never run for years now ... should we probably delete them or are they for any good?

**Comment by akurtakov** [comment]
tycho-deploy-dependencies was needed before platform uploaded ecj and equinox to maven central, no use of it now. Targeteditor and nexus are effectively dead.  tycho-build, tycho-gerrit, tycho-its are handled by tycho-github. I've removed all these now.

**Reply by laeubi** [reply]
Thanks!
